adhocracy4/test/helpers.py

```diff
@@ -33,3 +37,29 @@ def redirect_target(response):
 def render_template(string, context=None):
     context = Context(context or {})
     return Template(string).render(context)
+
+
+@contextmanager
+def skip_background_mail():
```

Comment:
the problem is the validation if the input is json serializable.
before i did implement a skip_decorator(decorator, module) context which replaced the given decorator used in the given module with an identity decorator. But after a review feedback it was decided to check that the input is serializable. although we were not 100% sure about it, as we define in the doc, that the input has to be serializable and we use internal knowledge of the thirdparty module by checking the serializibility with json.
what is your opinion on that? should we skip the serialization test and rely on the definition of valid input for the test?